Dockerfile

```diff
@@ -36,9 +37,24 @@ ENV RUST_TOOLCHAIN_VERSION=$RUST_TOOLCHAIN_VERSION
 ENV PATH="/usr/local/go/bin:$CARGO_HOME/bin:${PATH}"
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain "$RUST_TOOLCHAIN_VERSION"
 
+# Install Node.js
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+ENV NVM_DIR=/root/.nvm
+RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
+RUN . "$NVM_DIR/nvm.sh" && nvm use v${NODE_VERSION}
+RUN . "$NVM_DIR/nvm.sh" && nvm alias default v${NODE_VERSION}
+ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
+RUN npm i -g yarn
+
 # Install soroban-cli
 COPY --from=soroban-cli /usr/local/cargo/bin/soroban $CARGO_HOME/bin/
 
+# Install js-soroban-client
+ARG JS_SOROBAN_CLIENT_VERSION
```

Comment:
minor suggestion for consideration, include the type of repo in the arg name which the version refers such as `JS_SOROBAN_CLIENT_NPM_VERSION`